src/core/chart-registry.ts

```diff
@@ -96,9 +76,8 @@ class ChartRegistry {
      * @param {String} [group] Group name
      * @returns {Array<Object>}
      */
-    public list (group: string): BaseMixin[] {
-        group = this._initializeChartGroup(group);
-        return (this._chartMap)[group];
+    public list (group?: string): BaseMixin[] {
+        return this.chartGroup(group).list() as BaseMixin[];
```

Comment:
Does this convert correctly? `.list()` returns `IMinimalChart[]`

This method is probably unused internally.

Reply:
Yes, it should be changed.

Reply:
Changed in https://github.com/dc-js/dc.js/pull/1729/commits/d8d308b55745bcb72db6860613ed1807f5316c94